docs/farming-&-staking/staking/operators/register-operator.mdx

```diff
@@ -265,9 +264,10 @@ services:
       - "0.0.0.0:40333:40333/tcp"
     restart: unless-stopped
     command: [
-      "--chain", "gemini-3g",
+      "run",
+      "--chain", "gemini-3h",
       "--base-path", "/var/subspace",
-      "--port", "30333",
+      "--listen-on", "30333",
```

Comment:
Here and in multiple other places syntax is wrong, it is not just port anymore

Reply:
Yes, the `--port` has been replaced by `--listen-on`.  Perhaps some github quirk, just double checked that all --port commands were indeed replaced.

Reply:
I was talking about value, not the key name. The value is wrong.